TODO.md

```diff
@@ -0,0 +1,6 @@
+# TODO
+- [ ] Discuss splitting edge node provisioning, and edge application deployment into different labs
+    - EG; https://github.com/ahussey-redhat/device-edge-workshops focuses on the device deployment, and;
+    - https://github.com/redhat-na-ssa/rhte-edge-lab-sno focuses on application deployment
+    - This should reduce the overall time it takes to complete the full lab, as 90mins is not nearly enough time to; provision and configure edge clusters, and deploy applications
+- [ ] Add input field for student number. Use the input field to pre-populate student number in resources
```

Comment:
I've started, but just working out how I'm going to inject it into the various spots. I think it'll have to be done on the client side, rather than through Jekyll, because Jekyll doesn't do dynamic content